requirements.txt

```
@@ -2,7 +2,7 @@ geographiclib
 geojson
 h5netcdf
 jsonschema
-numpy>=1.14
+numpy<2
```

The minimum Python is still a requirement. We are adding an upper pin only.

```suggestion
numpy>=1.14,<2
```